src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+Title: Basic client usage - tutorial
+TODO:  Warning: Ubuntu release versions hardcoded
+       Include links to planned tutorials (tutorial-bundles.md, tutorial-constraints.md)
+
+# Basic client usage - tutorial
```

Comment:
I'd remove the "Client" and just go with Basic Usage. Maybe CLI usage since it's specific to that client in particular.

Reply:
Well, "client" is a concept that is well defined and used throughout the documentation. I also think "Basic usage" is just too vague. I think "CLI" is not a well enough known acronym to the intended audience.